packages/node_modules/@ciscospark/widget-roster/src/components/add-participant/index.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import classNames from 'classnames';
+import {
+  ICON_TYPE_EXIT
+} from '@ciscospark/react-component-icon';
+import Button from '@ciscospark/react-component-button';
+import Spinner from '@ciscospark/react-component-spinner';
+import PersonListItem from '@ciscospark/react-component-person-list-item';
+
+import styles from './styles.css';
+
+export default function AddParticipant(props) {
+  const baseClassPrefix = `ciscospark-roster-add-participant`;
+
+  function handleChange(e) {
+    props.onChange(e.target.value);
+  }
+
+  function handleClose() {
+    props.onChange(``);
+    props.onDismiss();
+  }
+
+  function handleClick(person) {
+    props.onAddPerson(person);
+  }
+
+  let results;
+  if (props.searchResults) {
+    if (props.searchResults.inProgress) {
+      // eslint-disable-next-line no-extra-parens
+      results = (
+        <div className={classNames(`${baseClassPrefix}-results-loading`, styles.resultsLoading)}>
+          <Spinner />
+        </div>
+      );
+    }
+    else if (props.searchResults.results && props.searchResults.results.length) {
+      // eslint-disable-next-line arrow-body-style
+      const listing = props.searchResults.results.map((person) => {
+        const onClick = handleClick.bind(this, person);
+        return (
+          // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+          <div
+            key={person.id}
+            onClick={onClick}
+            onKeyPress={onClick}
```

Comment:
May want to create a separate handler for this. Did you mean to fire of an Add Person with any keypress?